.github/workflows/main.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master, dev, dev-3.0.0, release/3.0.0 ]
+  pull_request:
+    branches: [ master, dev, dev-3.0.0, release/3.0.0 ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: windows-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+
+    - name: Install VirtoCommerce.GlobalTool
+      run: dotnet tool install --global VirtoCommerce.GlobalTool --version 3.0.0-beta0010
+
+    - name: Build
+      run: vc-build Compile
+
+    - name: Unit Tests
+      run: vc-build Test -skip Restore+Compile
+
+    - name: Build Package
+      run: vc-build Compress -skip Clean+Restore+Compile+Test
```

Comment:
you can leave only the `vc-build Compress `  it will execute all targets anyway.

Reply:
I know. I split it into independent steps to visualize progress.